src/toolbox/utils/return-untouched.spec.ts

```diff
@@ -0,0 +1,15 @@
+import { returnUntouched } from "./return-untouched";
+import { expect } from "chai";
+import "mocha";
+
+const tests = [[54321, 54321], ["abc", "abc"]];
```

Comment:
What about other kinds of values, like booleans, objects, arrays, nulls, etcs.. ?